checker.js

```diff
@@ -1,32 +1,66 @@
-function batteryIsOk(temperature,  soc,  chargeRate) {
-    if(temperature < 0 || temperature > 45) {
-        console.log("Temperature is out of range!");
-        return false;
-    } else if(soc < 20 || soc > 80) {
-        console.log("State of Charge is out of range!");
-        return false;
-    } else if(chargeRate > 0.8) {
-        console.log("Charge Rate is out of range!");
-        return false;
-    }
-    return true;
+const LOW_TEMP = 0
+const HIGH_TEMP = 45
+const LOW_SOC = 20
+const HIGH_SOC = 80
+const HIGH_CHARGE_RATE = 0.8
+
+function batteryIsOk(temperature, soc, chargeRate) {
+
```

Comment:
It appears that DRY principle is violated because every parameter check function repeats if/else range check.